From 6edfe5cc03b24df9f3359657ee43b63265f53dff Mon Sep 17 00:00:00 2001
From: Patrick Yavitz <pyavitz@gmail.com>
Date: Sat, 24 Jan 2026 08:43:12 -0500
Subject: [PATCH] mmc: sdio: Don't warn about unknown CIS tuples

[   12.036570] mmc2: queuing unknown CIS tuple 0x01 [d9 01 ff] (3 bytes)
[   12.048163] mmc2: queuing unknown CIS tuple 0x1a [01 01 00 02 07] (5 bytes)
[   12.053004] mmc2: queuing unknown CIS tuple 0x1b [c1 41 30 30 ff ff 32 00] (8 bytes)
[   12.053973] mmc2: queuing unknown CIS tuple 0x14 [] (0 bytes)

Signed-off-by: Patrick Yavitz <pyavitz@gmail.com>
---
 drivers/mmc/core/sdio_cis.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/mmc/core/sdio_cis.c b/drivers/mmc/core/sdio_cis.c
index afaa6cab1adc..dabfc6be4bba 100644
--- a/drivers/mmc/core/sdio_cis.c
+++ b/drivers/mmc/core/sdio_cis.c
@@ -345,7 +345,7 @@ static int sdio_read_cis(struct mmc_card *card, struct sdio_func *func)
 						tpl_code, tpl_link, this->data,
 						tpl_link);
 				else
-					pr_warn_ratelimited(FMT("unknown"),
+					pr_debug_ratelimited(FMT("unknown"),
 						mmc_hostname(card->host),
 						tpl_code, tpl_link, this->data,
 						tpl_link);
-- 
2.51.0

